tensilelite/Tensile/TensileInstructions/Code.py

```diff
@@ -680,7 +680,7 @@ def __str__(self):
         kStr += kdIndent + ".amdhsa_system_vgpr_workitem_id %u\n" % self.vgprWorkItem
         kStr += kdIndent + ".amdhsa_float_denorm_mode_32 3\n"
         kStr += kdIndent + ".amdhsa_float_denorm_mode_16_64 3\n"
-        if self.enablePreloadKernArgs:
+        if self.enablePreloadKernArgs and (self.kernel.isa == (9, 0, 10) or self.kernel.isa == (9, 4)):
```

Comment:
Should disable "PreloadKernArgs" in SolutionStruct.py directly, but not here.

Reply:
done